src/compiler/emitter.ts

```diff
@@ -4707,6 +4707,9 @@ if (typeof __param !== "function") __param = function (paramIndex, decorator) {
                 }
 
                 write("[\"require\", \"exports\"");
+                if (compilerOptions.noEmitHelpers) {

```

Comment:
We are considering a different approach to making `__extends` and other helpers available with something like `--noEmitHelpers`, so I would recommend removing this line.